public/stylesheets/setting.css

```diff
@@ -0,0 +1,36 @@
+body {
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  height: 100dvh;
+  background-image: url(../images/background-image.png);
+  background-size: cover;
+  background-position: center;
+}
+
+h2 {
+  /*テキストをドラッグできないようにします*/
+  -ms-user-select: none;
+  -webkit-user-select: none;
+  user-select: none;
+
+  font-size: 60px;
+  text-align: center;
+  padding-top: 40px;
+}
+
+p {
+  /*テキストをドラッグできないようにします*/
+  -ms-user-select: none;
+  -webkit-user-select: none;
+  user-select: none;
+
+  /* 水平方向の中央揃え */
+  font-size: 50px;
+  text-align: center;
+}
+
+input {
+  transform: translate(50%, 50%);
+  width: 50%;
+}
```

Comment:
Mackbookなどの端末だと高さが足りないのでうまく調節し直しました。
これとHTMLの方のsuggestionを取り込んで、ドラッグでの文字選択の禁止や余白・文字サイズを修正して再度pushしてください。

<img width="721" alt="image" src="https://github.com/user-attachments/assets/4d14793c-98c9-4447-a4ba-4ba9e28720ca">


```suggestion
[data-screen="setting"] {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  gap: 2rem;
  width: 100%;
  height: 100%;
}

.volume-control {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  width: 100%;
}

.volume-control input[type="range"] {
  width: 50%;
}
```